Sunflower/build.gradle

```diff
@@ -18,38 +18,38 @@ buildscript {
     // Define versions in a single place
     ext {
         // Sdk and tools
-        compileSdkVersion = 31
+        compileSdkVersion = 33
```

Comment:
the sdks compile against 32 - do we need to have the sample app at the same version?

Reply:
I originally updated to 32, but then one of the updated Android libraries required compiling with 33.